src/main/java/bowling/score/Score.java

```diff
@@ -18,34 +23,53 @@ private Score(int value, int bonusCount) {
 		this.bonusCount = bonusCount;
 	}
 
-	public static Score of(int value, int bonusCount) {
-		return new Score(value, bonusCount);
+	public static Score unavailable() {
+		return new Score(UNAVAILABLE_SCORE, UNAVAILABLE_COUNT);
+	}
+
+	public static Score remain(Score first) {
+		return Score.of(first.value, MIN_BONUS_COUNT);
 	}
 
 	public static Score open(Score first, Score second) {
-		return new Score(first.value + second.value, OPEN_BONUS_COUNT);
+		return Score.of(first.value + second.value, MIN_BONUS_COUNT);
 	}
 
-	public static Score spare(Score maxScore) {
-		return new Score(maxScore.value, SPARE_BONUS_COUNT);
+	public static Score spare(Score score) {
+		return Score.of(score.value, SPARE_BONUS_COUNT);
 	}
 
 	public static Score strike(Score score) {
-		return new Score(score.value, STRIKE_BONUS_COUNT);
+		return Score.of(score.value, STRIKE_BONUS_COUNT);
+	}
+
+	public static Score of(int value, int bonusCount) {
+		Validator.min(MIN_SCORE, value,
+			String.format("점수의 최솟값(%d) 보다 입력(%d)이 작습니다.", MIN_SCORE, value));
+		Validator.min(MIN_BONUS_COUNT, bonusCount,
+			String.format("보너스 카운트 최솟값(%d) 보다 입력(%d)이 작습니다.", MIN_BONUS_COUNT, value));
+		return new Score(value, bonusCount);
 	}
 
 	public boolean canScore() {
+		if (!isAvailable()) {
+			return false;
+		}
 		return bonusCount == 0;
```

Comment:
```suggestion
		if (isAvailable()) {
			return bonusCount == 0;
		}
		return false;
```
사소하지만 개인적으로는 좀 더 가독성에 좋다고 생각합니다 :)